Dockerfile

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright 2019-present Open Networking Foundation
+# Copyright (c) 2003-2018, Great Software Laboratory Pvt. Ltd.
+# Copyright (c) 2017 Intel Corporation
```

Comment:
The copyright date should reflect the date of first publication and dates in which major changes were published. 
So this is the first version of the file and created in 2019, we should only keep the ONF line here, and delete the other 2 copyright lines. 
Later on, when other company modify this file, they will put their own header on top of ONF.

Reply:
OK, thanks!